controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/Policy.java

```diff
@@ -164,6 +164,12 @@
             .on(PathGroup.billingPlan)
             .in(SystemName.PublicCd)),
 
+    /** Ability to update tenant collection method */
+    collectionMethodUpdate(Privilege.grant(Action.update)
+            .on(PathGroup.billingCollection)
+            .in(SystemName.PublicCd)),
```

Comment:
Just make this for all of public now?

Reply:
We could - I suggest we make a PR where we change all relevant policies in one go